lint-configs/.yamllint.yml

```
@@ -0,0 +1,60 @@
+# Created based on v1.35.1
+# NOTE:
+# - Rules are only configured if they differ from the default configuration.
+# - Users should use `yamllint` with the `--strict` option to treat warnings as errors.
+
+extends: "default"
+
+yaml-files:
+  - ".clang-format"
+  - ".yamllint"
+  - "*.yaml"
+  - "*.yml"
+
+rules:
+  anchors:
+    # Disabled so that we can reference aliases that exist later in the file
+    forbid-undeclared-aliases: false
+
+  braces:
+    min-spaces-inside-empty: 0
+    max-spaces-inside-empty: 0
```

Do we want to ensure alpha-beta ordering?
```suggestion
    max-spaces-inside-empty: 0
    min-spaces-inside-empty: 0
```

---

Good idea.